Resources/skeleton/web/config.php

```diff
@@ -12,7 +12,7 @@
     exit('This script is only accessible from localhost.');
 }
 
-require_once dirname(__FILE__).'/../app/SymfonyRequirements.php';
+require_once __DIR__.'/../app/SymfonyRequirements.php';
```

Comment:
this file is copied in the symfony-standard skeleton. and people can download the SE archive from symfony.com rather than using the symfony installer or composer for the initial setup (not sure whether this option is still available on the website though)